### PR TITLE
Database

### PR DESCRIPTION
**Pull request by mtracy01**
I filled out the information for Reports.  Look good?

### CODE REVIEW

File: dli_app/mod_reports/models.py [hunk 1]
```diff
@@ -15,8 +15,9 @@
 class Report(db.Model):
     __tablename__ = "report"
     id = db.Column(db.Integer, primary_key=True)
-    # TODO: Fill out remaining fields
-    fields = db.relationship(
+    user_id = db.Column(db.Integer, index=True)
+	rep_name = db.Column(db.String(64))
```
Comment (gorel) [inline dli_app/mod_reports/models.py]: s/rep_name/name/

Comment (gorel) [inline dli_app/mod_reports/models.py]: Also, is that just randomly tabbed over?

Comment (gorel) [inline dli_app/mod_reports/models.py]: YOU MONSTER. YOU'RE USING TAB LITERALS.

File: dli_app/mod_reports/models.py [hunk 2]
```diff
@@ -28,18 +29,16 @@ class Report(db.Model):
...
+        return '<Report %r>' % self.rep_name
 
 class Field(db.Model):
     __tablename__ = "field"
     id = db.Column(db.Integer, primary_key=True)
     field_type = db.relationship('FieldType')
-    # TODO: Fill out remaining fields
-
+	department_id = db.Column(db.Integer, db.ForeignKey("department.id"))
+	name = db.Column(db.String(32))
```
Comment (gorel) [inline dli_app/mod_reports/models.py:82]: Add blank line before function definition

Comment (gorel) [inline dli_app/mod_reports/models.py:82]: One more thing to add to this (for easier retrieval within code):
data_points = db.relationship(
    'FieldData',
    backref='field',
)

File: dli_app/mod_reports/models.py [hunk 3]
```diff
@@ -15,7 +15,8 @@
 class Report(db.Model):
     __tablename__ = "report"
     id = db.Column(db.Integer, primary_key=True)
-    # TODO: Fill out remaining fields
+    user_id = db.Column(db.Integer, index=True)
+    rep_name = db.Column(db.String(64))
```
Comment (gorel) [inline dli_app/mod_reports/models.py]: s/rep_name/name

File: dli_app/mod_reports/models.py [hunk 4]
```diff
@@ -52,11 +51,13 @@ def __repr__(self):
...
-
+    date_stamp = db.Column(db.DateTime, primary_key=True)
+    field_id = db.Column(db.Integer, db.ForeignKey("field.id"))
+    ivalue = db.Column(db.BigInteger)
+    dvalue = db.Column(db.Double)
+    svalue = db.Column(db.String(128))
     def __repr__(self):
-        # TODO: Find a way of describing this field data
-        return '<FieldData>'
+        return '<FieldData %r>' % svalue
```
Comment (gorel) [inline dli_app/mod_reports/models.py]: self.svalue

File: dli_app/mod_reports/models.py [hunk 5]
```diff
@@ -28,18 +29,16 @@ class Report(db.Model):
...
+        return '<Report %r>' % self.rep_name
 
 class Field(db.Model):
     __tablename__ = "field"
     id = db.Column(db.Integer, primary_key=True)
     field_type = db.relationship('FieldType')
-    # TODO: Fill out remaining fields
-
+    department_id = db.Column(db.Integer, db.ForeignKey("department.id"))
+    name = db.Column(db.String(32))
```
Comment (gorel) [inline dli_app/mod_reports/models.py]: One more thing to add to this (for easier retrieval within code):
data_points = db.relationship(
'FieldData',
backref='field',
)

Comment (gorel) [inline dli_app/mod_reports/models.py]: Add line before function definition

**Comment by faultyserver** [comment]
Is this ready to get merged yet? It has a lot of the same fixes that I've needed to make for UI stuff, so I'd like to be able to rebase onto this and avoid re-committing the same changes.

**Comment by gorel** [comment]
Almost, but not quite. Let me make like 2 more minor nit-pick suggestions.

**Comment by gorel** [comment]
Fixed all the merge conflicts. Gonna go ahead and merge this.

**Pull request merged**